pytorch_lightning/trainer/data_loading.py

```diff
@@ -37,12 +37,12 @@ class TrainerDataLoadingMixin(ABC):
     # this is just a summary on variables used in this abstract class,
     #  the proper values/initialisation should be done in child class
     global_rank: int
-    shown_warnings:...
+    shown_warnings: ...
```

Comment:
@akihironitta thinking that if we replace the elliptic placeholder with ANy it shall be fine, right?
```suggestion
    shown_warnings: Any
```